httpasyncclient/src/main/java/io/opentracing/contrib/httpcomponents/SpanHttpAsyncClient.java

```diff
@@ -20,12 +20,22 @@
     private final Tracer tracer;
     private final ContextSpan contextSpan;
     private final HttpTaggerFactory taggerFactory;
+    private final SpanModifier spanModifier;
 
     public SpanHttpAsyncClient(final CloseableHttpAsyncClient client, final Tracer tracer, final ContextSpan contextSpan, final HttpTaggerFactory taggerFactory) {
         this.client = client;
         this.tracer = tracer;
         this.contextSpan = contextSpan;
         this.taggerFactory = taggerFactory;
+        this.spanModifier = NoOpSpanModifier.INSTANCE;
```

Comment:
Could this just call the other constructor?

```suggestion
        this(client, tracer, contextSpan, taggerFactory, spanModifier, NoOpSpanModifier.INSTANCE);
```